src/lib/components/Accordion/AccordionItem.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import { getContext, onMount } from "svelte";
+  import type { AccordionContext, AccordionContextItems } from "./types";
+
+  // Note: `id` has a fallback value, but ideally,
+  // one should supply their own identifiers
+  export let id = "accordionItem-" + Math.random().toString(36);
+  export let title: string | null | undefined = "Title";
+  export let expanded = false;
+
+  const context: AccordionContext = getContext("Accordion");
```

Comment:
is that a concern since it's via Context? I.e., since `AccordionItem`s should always be getting the context of their parent `Accordion`, is there an actual danger of that?